### DISCUSSION TITLE
have g14 - anime lid display stopped working...any ideas?

### DISCUSSION THREAD

**Discussion by deepmenace** [seerge/g-helper#721] [Q&A]
hi all, has anyone with a g14 had their anime matrix display stop working and managed to fix it? a combo of drivers to install maybe? it stopped appearing as an option in armour crate - i thought that maybe installing ghelper might resurrect it, but no...

**Comment by seerge** [comment]
@deepmenace try to reset it https://github.com/seerge/g-helper#how-do-i-do-a-hardware-reset-on-a-laptop

**Reply by deepmenace** [reply]
i still cant see anything in ghelper related to animatrix...it has made the internet faster though i think :-D haha - wifi seems snappier. 

there are reports of this happening on reddit and people fixed it reloading a load of drivers in a certain order - it just hasnt worked for me .

**Reply by seerge** [reply]
Well, you can try to delete / reinstall USB devices under device manager. Anime matrix is one of them, but i don't know which one specifically.
